### DISCUSSION TITLE
encoding geospatial reference in qr code / utilizing the gps in tablets

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1063] [users: ideas for new features]
- since the ipads can do it, even though with quite some inaccuracy, I suspect  ... would that make any sense? 
- would it make sense to use the gps for those projects which do not have gis or a proper surveyor? 
- can one do elevations with that?

**Comment by luizaogs** [comment]
Don't ask me why this came to mind when I couldn't sleep last night because I wouldn't have an answer for you, but this might be more useful for survey than excavation? The utilizing the gps in tablets, I mean. For both transect and grabbing types. One should use a map but it might be useful for specific findspots of surface finds/features? 

(I don't think it would work for elevations? But could be wrong.)

**Comment by lbestock** [comment]
It will never make sense in an excavation unit because there the resolution needed is always too high, whether for elevation or XY. But for survey and for features it does make sense. But you ask it here as linked to the QR code and that I want to discuss. One might want those spatial data without a photo. For a feature, a photo with spatial data would be really pretty great. On survey, the useful points to record might include photos, but would also include the beginning and end point of each transect. If we could get spatial data on those things we would be able to do survey even without the RTKs, and having the basic start and end point on each transect would in fact be more accurate than how we record survey even when we do use the RTKs to map a survey unit and then just approximate the 10m between transect walkers.

**Comment by lbestock** [comment]
See #1240 for a way survey transects get used that we don't yet support but that would make sense to think of in combination with the gps in an iPad.

**Comment by urapadmin** [comment]
@nparkervan: And you might be interested in joining this discussion
